source/includes/custom-field-options/_update.md.erb

```diff
@@ -0,0 +1,58 @@
+## Update Custom Field Option
+
+```http
+PATCH https://api.teamtailor.com/v1/custom-field-options/1 HTTP/1.1
+Authorization: Token token=abc123abc123
+X-Api-Version: <%= config[:api_version] %>
+
+{
+  "data": {
+    "id": 1,
+    "type": "custom-field-options",
+    "attributes": {
+      "value": "Updated value",
+    }
+  }
+}
+```
+
+```shell
+curl -X "PATCH" "https://api.teamtailor.com/v1/custom-field-options/1" \
+     -H "Authorization: Token token=abc123abc123" \
+     -H "X-Api-Version: <%= config[:api_version] %>"
+     -H "Content-Type: application/vnd.api+json" \
+     -d '{
+   "data": {
+     "id": 1,
+     "type": "custom-field-options",
+     "attributes": {
+       "value": "Updated value"
+     }
+   }
+}'
+```
+
+> Example response
+
+```json
+{
+  "data": {
+    "id": "1",
+    "type": "custom-field-options",
+    "links": {
+      "self": "https://api.teamtailor.localhost/v1/custom-field-options/1"
+    },
+    "attributes": {
+      "value": "Updated value"
+    },
+    "relationships": {
+      "custom-field": {
+        "links": {
+          "self": "https://api.teamtailor.localhost/v1/custom-field-options/1/relationships/custom-field",
+          "related": "https://api.teamtailor.localhost/v1/custom-field-options/1/custom-field"
```

Comment:
```suggestion
      "self": "https://api.teamtailor.com/v1/custom-field-options/1"
    },
    "attributes": {
      "value": "Updated value"
    },
    "relationships": {
      "custom-field": {
        "links": {
          "self": "https://api.teamtailor.com/v1/custom-field-options/1/relationships/custom-field",
          "related": "https://api.teamtailor.com/v1/custom-field-options/1/custom-field"
```

Reply:
Checked it multiple times! forgot some anyways 🙁